### PR TITLE
Add API docs for backend services

### PR DESCRIPTION
**Pull request by m-orfanos**
Add API docs for backend services (list, get, add)

### CODE REVIEW

File: source/includes/gcp/_backend_services.md [hunk 1]
```diff
@@ -1 +1,98 @@
...
+| `name`<br/>*string*                 | Name of the resource. |
+| `port`<br/>*number*                 | Deprecated in favor of portName. The TCP port to connect on the backend. The default value is 80. |
+| `portName`<br/>*string*             | A named port on a backend instance group representing the port for communication to the backend VMs in that group. |
+| `protocol`<br/>*enum*               | The protocol this BackendService uses to communicate with backends. Possible values are HTTP, HTTPS, HTTP2, TCP, SSL, or UDP. |
+| `selfLink`<br/>*string*             | Server-defined URL for the resource. |
+| `sessionAffinity`<br/>*enum*        | Type of session affinity to use. The default is NONE. |
+| `shortBackends`<br/>*string*        | A short version of the list of backends. |
+| `shortHealthChecks`<br/>*string*    | A short version of the health checks. |
+| `timeoutSec`<br/>*number*           | This field has a different meaning depending on the type of load balancer. See [here](https://cloud.google.com/load-balancing/docs/backend-service#backend_service_settings) The default is 30 seconds. |
```
Comment (SeventhOfNine) [inline source/includes/gcp/_backend_services.md]: ideally we don't refer to google's docs. I would just put the last sentence.

File: source/includes/gcp/_backend_services.md [hunk 2]
```diff
@@ -1 +1,98 @@
...
+| `healthChecks`<br/>*string*         | The list of URLs to the HttpHealthCheck or HttpsHealthCheck resource for health checking this BackendService. Currently at most one health check can be specified. |
+| `id`<br/>*string*                   | The unique identifier for the resource. This identifier is defined by the server. |
+| `kind`<br/>*string*                 | Type of resource. Always compute#backendService for backend services.|
+| `loadBalancingScheme`<br/>*enum*    | Indicates whether the backend service will be used with internal or external load balancing. Possible values are INTERNAL and EXTERNAL.|
+| `name`<br/>*string*                 | Name of the resource. |
+| `port`<br/>*number*                 | Deprecated in favor of portName. The TCP port to connect on the backend. The default value is 80. |
+| `portName`<br/>*string*             | A named port on a backend instance group representing the port for communication to the backend VMs in that group. |
+| `protocol`<br/>*enum*               | The protocol this BackendService uses to communicate with backends. Possible values are HTTP, HTTPS, HTTP2, TCP, SSL, or UDP. |
```
Comment (SeventhOfNine) [inline source/includes/gcp/_backend_services.md]: Remember we only support HTTP and HTTPS at the moment, so remove the others.

File: source/includes/gcp/_backend_services.md [hunk 3]
```diff
@@ -1 +1,98 @@
...
+| `backends`<br/>*object*             | The list of backends that serve this BackendService.|
+| `connectionDraining`<br/>*object*   | - |
+| `creationTimestamp`<br/>*string*    | Creation timestamp in RFC3339 text format. |
+| `description`<br/>*string*          | An optional description of this resource. Provide this property when you create the resource.|
+| `enableCDN`<br/>*boolean*           | If true, enables Cloud CDN for the backend service. Only applicable if the `loadBalancingScheme` is EXTERNAL and the protocol is HTTP or HTTPS. |
+| `fingerprint`<br/>*string*          | Fingerprint of this resource. A hash of the contents stored in this object. |
+| `healthChecks`<br/>*string*         | The list of URLs to the HttpHealthCheck or HttpsHealthCheck resource for health checking this BackendService. Currently at most one health check can be specified. |
+| `id`<br/>*string*                   | The unique identifier for the resource. This identifier is defined by the server. |
+| `kind`<br/>*string*                 | Type of resource. Always compute#backendService for backend services.|
```
Comment (SeventhOfNine) [inline source/includes/gcp/_backend_services.md]: Type of the resource

File: source/includes/gcp/_backend_services.md [hunk 4]
```diff
@@ -1 +1,98 @@
...
+
+Retrieve a list of all backend services in an [environment](#administration-environments).
+
+| Attributes                          | &nbsp; |
+| ----------------------------------- | ------ |
+| `affinityCookieTtlSec`<br/>*number* | If set to 0, the cookie is non-persistent and lasts only until the end of the browser session (or equivalent). The maximum allowed value is one day (86,400).|
+| `backends`<br/>*object*             | The list of backends that serve this BackendService.|
+| `connectionDraining`<br/>*object*   | - |
+| `creationTimestamp`<br/>*string*    | Creation timestamp in RFC3339 text format. |
+| `description`<br/>*string*          | An optional description of this resource. Provide this property when you create the resource.|
```
Comment (SeventhOfNine) [inline source/includes/gcp/_backend_services.md]: An optional description of this resource

File: source/includes/gcp/_backend_services.md [hunk 5]
```diff
@@ -1 +1,98 @@
...
+| `enableCDN`<br/>*boolean*           | If true, enables Cloud CDN for the backend service. Only applicable if the `loadBalancingScheme` is EXTERNAL and the protocol is HTTP or HTTPS. |
+| `fingerprint`<br/>*string*          | Fingerprint of this resource. A hash of the contents stored in this object. |
+| `healthChecks`<br/>*string*         | The list of URLs to the HttpHealthCheck or HttpsHealthCheck resource for health checking this BackendService. Currently at most one health check can be specified. |
+| `id`<br/>*string*                   | The unique identifier for the resource. This identifier is defined by the server. |
```
Comment (SeventhOfNine) [inline source/includes/gcp/_backend_services.md]: The id of the backend service
We usually say it's a UUID not a string

File: source/includes/gcp/_backend_services.md [hunk 6]
```diff
@@ -1 +1,98 @@
...
+| `kind`<br/>*string*                 | Type of resource. Always compute#backendService for backend services.|
+| `loadBalancingScheme`<br/>*enum*    | Indicates whether the backend service will be used with internal or external load balancing. Possible values are INTERNAL and EXTERNAL.|
+| `name`<br/>*string*                 | Name of the resource. |
+| `port`<br/>*number*                 | Deprecated in favor of portName. The TCP port to connect on the backend. The default value is 80. |
```
Comment (SeventhOfNine) [inline source/includes/gcp/_backend_services.md]: I would just put: The TCP port to connect on the backend. The default value is 80
Or remove it, I am not sure we want to document deprecated stuff, even if google does (well they have to it's their API)

Comment (m-orfanos) [reply source/includes/gcp/_backend_services.md]: I removed it.

**Comment by m-orfanos** [comment]
CR suggestions done.

**Review by simongodard — APPROVED** [review]

**Review by jasondias9 — APPROVED** [review]

**Pull request merged**